src/sinks/honeycomb.rs

```diff
@@ -18,10 +17,11 @@ use crate::{
     },
 };
 
-static HOST: Lazy<Uri> = Lazy::new(|| Uri::from_static("https://api.honeycomb.io/1/batch"));
-
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub(super) struct HoneycombConfig {
+    #[serde(skip, default = "default_endpoint")]
+    endpoint: String,
+
```

Comment:
Ditto needs documentation.

Reply:
It's marked `skip` and always has a default, so it's not part of the user-facing configuration.